src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -238,13 +244,13 @@ public void onTraversalStart() {
         if (!intervalArgumentCollection.intervalsSpecified()) {
             throw new UserException("you must indicate the set of exon as input intervals");
         }
-        genomeLocFactory = new GenomeLocParser(sequenceDictionary);
         sampleCollection = new SampleCollection(getHeaderForReads());
 
         logger.log(Level.INFO, "Reading exons locations from intervals...");
-        final List<GenomeLoc> locations = IntervalUtils.genomeLocsFromLocatables(genomeLocFactory,
-                intervalArgumentCollection.getIntervals(sequenceDictionary));
-        exonCollection = new IntervalBackedExonCollection(locations);
+
+        exonCollection = new IntervalBackedExonCollection(
+                intervalArgumentCollection.getIntervals(sequenceDictionary),
+                (contig1,contig2) -> Integer.compare(sequenceDictionary.getSequenceIndex(contig1),sequenceDictionary.getSequenceIndex(contig2)));
```

Comment:
This is clever, but why are you specifying the comparator here?  Is there ever a case when you would want a different comparator?